src/libs/httpserver/httpCookieManager.ts

```diff
@@ -75,10 +82,14 @@ const parseCookie = cookieString => {
  * @param {CozyClient} client - CozyClient instance
  * @returns {Promise<boolean>}
  */
-export const setCookie = async (cookieString, client) => {
+export const setCookie = async (
```

Comment:
These functions seem interesting to test (only the exported ones)

Reply:
Those tests already exist in: https://github.com/cozy/cozy-flagship-app/blob/master/src/libs/httpserver/httpCookieManager.spec.js

Is that what you are talking about?

Reply:
Yea my bad!